allauth/socialaccount/providers/trainingpeaks/views.py

```diff
@@ -0,0 +1,59 @@
+import requests
+
+from allauth.socialaccount import app_settings
+from allauth.socialaccount.providers.oauth2.views import (
+    OAuth2Adapter,
+    OAuth2CallbackView,
+    OAuth2LoginView,
+)
+
+from .provider import TrainingPeaksProvider
+
+
+class TrainingPeaksOauth2Adapter(OAuth2Adapter):
```

Comment:
Can you please rename Oauth to OAuth ?

Reply:
Nice one! Fixed and rebased branch with Master, should merge cleanly now.

Thanks!